app/javascript/src/components/box-assembly-form/BoxAssemblyForm.jsx

```diff
@@ -5,6 +5,8 @@ import './BoxAssemblyForm.scss';
 const BoxAssemblyForm = () => {
   const [items, updateItems] = useState(null);
   const [isAssembled, updateIsAssembled] = useState(false);
+  const [selectedLocation, setLocation] = useState(null);
+  const [selectedInventoryTally, setSelectedInventoryTall] = useState({});
```

Comment:
Should this be `setSelectedTally` ?

Reply:
Yeah that makes sense, I'll update

Reply:
Actually that got taken out in a future commit: 50a926cb39b57f7c7032cc9d4d69af0b01c58567 by commenting that line out. I've pushed a new commit that removes the comment all together.